LayoutTests/fast/frames/consume_transient_activation.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <script src="../../resources/js-test-pre.js"></script>
+    <script>
+      description("Test consuming transient activation.");
+
+      var frame;
+      var innerFrame;
+      var innerFrame2;
+
+      function dispatchClick() {
+        var x = frame.offsetLeft + frame.offsetWidth / 2;
+        var y = frame.offsetTop + frame.offsetHeight / 2;
+
+        eventSender.mouseMoveTo(x, y);
```

Comment:
We should probably be using activateAt() from ui-helper.js so that the test can be cross-platform.